### ISSUE TITLE
evolve and Postgres get error "SHOW VARIABLES LIKE '"

### ISSUE THREAD

**Issue by ohadab29** [lecaillon/Evolve#307]
Hi all ,

using Postgres DB and Evolve i saw i cam e cross with this ERROR : 
 2023-07-11 05:45:08.393 UTC [103] ERROR:  syntax error at or near "LIKE" at character 16
vulnerability-notifier-database-1     | 2023-07-11 05:45:08.393 UTC [103] STATEMENT:  SHOW VARIABLES LIKE '%version%'

at the end i understand Evolve libraary is trying to run query on the DB but syntax is not Postgress....

do you have a solution for that not putting too much noise to the logs?

similar old case i saw : 👍 https://github.com/lecaillon/Evolve/issues/70

Thanks
Ohad

**Comment by lecaillon** [comment]
Yes you can. It is not an issue. Evolve try to determinate the type of db you use at start. If you do not want that message, pass the information in the constructor : 

```
var evolve = new Evolve(cnn, msg => Output.WriteLine(msg), DBMS.PostgreSQL)
{
    Schemas = new[] { "public", "unittest" },
    MetadataTableSchema = "unittest",
    Placeholders = new Dictionary<string, string> { ["${schema1}"] = "unittest" }
};
```

**Comment by ohadab29** [comment]
will try it - thank u !!!

**Comment by ohadab29** [comment]
today i use : 
  {
                var evolve = new Evolve.Evolve((DbConnection) connection)
                {
                    EmbeddedResourceAssemblies = new[] {GetType().Assembly},
                    SqlMigrationPrefix = "v",
                    MetadataTableSchema = "pprepository"

                };

what in the syntax u mentioned will remove the log of error ?

**Comment by lecaillon** [comment]
`DBMS.PostgreSQL`
